google/cloud/bigtable/client_options.h

```diff
@@ -37,15 +37,26 @@ class ClientOptions {
   /**
    * Initialize the client options with the default credentials.
    *
-   * If the `BIGTABLE_EMULATOR_HOST` environment variable is set this
-   * constructor configures the object to connect to the host and port set in
-   * that environment variable. If the environment variable is not set, this
-   * constructor configures the object to connect to the production instance of
-   * Cloud Bigtable using the application default credentials.
+   * Configure the client to connect to the production instance of Cloud
+   * Bigtable, using the default google credentials for authentication.
+   *
+   * @par Environment Variables
+   * If the `BIGTABLE_EMULATOR_HOST` environment variable is set the default
```

Comment:
s/is set the default/is set, the default/

Reply:
Done